core/domain/skill_services_test.py

```diff
@@ -323,6 +378,44 @@ def test_get_rubrics_of_linked_skills(self) -> None:
             }
         )
 
+    def test_get_rubrics_skips_nonevalue_skills(self) -> None:
+        skill_1 = self.save_new_skill(
+            'skill_id_1', self.user_id_admin, description='Description 1',
+            misconceptions=[],
+            skill_contents=None,
+            rubrics=[
+                skill_domain.Rubric(
+                    constants.SKILL_DIFFICULTIES[0], ['Explanation 1']),
+                skill_domain.Rubric(
+                    constants.SKILL_DIFFICULTIES[1], ['Explanation 2']),
+                skill_domain.Rubric(
+                    constants.SKILL_DIFFICULTIES[2], ['Explanation 3']),
+            ]
+        )
+        with self.swap(skill_fetchers, 'get_multi_skills', lambda skill_ids, strict=False: [None, skill_1]):
+            skill_rubrics, _ = skill_services.get_rubrics_of_skills(['skill_id_1'])
+            self.assertEqual(
+                skill_rubrics, {
+                    'skill_id_1': [
+                        skill_domain.Rubric(
+                            constants.SKILL_DIFFICULTIES[0], ['Explanation 1']
+                        ).to_dict(),
```

Comment:
Instead of using to_dict, you can have the exact dictionary. Mainly in tests we want everything to be explicit.